### DISCUSSION TITLE
Multichannel streaming not providing separate channel transcriptions

### DISCUSSION THREAD

**Discussion by stri8ed** [deepgram/community#245] [General help] [answered]
I am creating a transcription live connection like so:

```
transcription.live({
            language: "en-US",
            model: "phonecall",
            Tier: "nova",
            multichannel: true,
            channels: 2,
            sample_rate: 8000,
            encoding: "mulaw"
            ...
        })
```
Based on the docs, the transcription data should include an array field `channels`, which contains data for each channel.

The responses that I receive via the WebSocket `transcriptReceived` event, do not not include a `channels` field. It only contains a `channel` field, which contains transcription for both channels (with no indication of which channel it belongs to). It also contains a `channel_index` field, which is an array containing two integers. That field is not documented anywhere on your site.

How can I obtains separate transcriptions for each channel, when streaming the data?

**Comment by nikolawhallon** [comment, marked as answer]
The `channel_index` field should be the key for this - the second number is the number of channels (should be "2" here), and the first number is the channel that the transcript in the message belongs to, so you should see messages with `[0, 2]` for transcription of the 1st (left) channel, and `[1, 2]` for transcription of the second (right) channel.

**Reply by shirgoldbird** [reply]
@stri8ed Thanks for pointing this out—we'll be updating our docs shortly to clarify the difference between pre-recorded multichannel transcription and streaming multichannel transcription!
